backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/CassandraTableManager.java

```diff
@@ -58,14 +62,23 @@ public InitializationStatus initializeTables() {
 
     public void clearAllTables() {
         CassandraAsyncExecutor executor = new CassandraAsyncExecutor(session);
-        Flux.fromIterable(module.moduleTables())
+        Flux.fromIterable(listAllTables(executor))
                 .publishOn(Schedulers.elastic())
-                .map(CassandraTable::getName)
                 .flatMap(name -> truncate(executor, name))
                 .then()
                 .block();
     }
 
+    private List<String> listAllTables(CassandraAsyncExecutor executor) {
```

Comment:
We should not change the default behavior of the Cassandra extension.
We can, for now, either generate a new keyspace for each smoketest or recreate the keyspace between two runs.